### PR TITLE
finsihed week-3

### PR DESCRIPTION
**Pull request by MichaelReyes1**
# Submitting Week 3 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ x] Behavioral Questions
- [ x] Career Readiness
- [ x] Coding Challenges
- [ x] Interview Questions
- [ x] React Debugging

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:
feel like im getting better other than remembering vocabulary, but im working on that on the daily and should start kicking soon . i ran into trouble with the debugging but realized it was supposed to start the page blank and once i made changes it started to work. i really liked that helped me understand it so much better!

### CODE REVIEW

File: career-readiness.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Creating a technical portfolio website is a big undertaking. Portfolios are almo
 
 Wire frames are used to outline the basic structure of your site. Wire frames should be vague enough to give you flexibility in your development process but structured enough to ensure all appropriate elements are present. 
 
-Submit the link to your wire frames:
+Submit the link to your wire frames: https://www.figma.com/file/nBenpfchFiXVljIU4m0dw6/Portfolio?type=design&node-id=0%3A1&mode=design&t=0Liek3YfLjrY7mV0-1
```
Comment (chrisyhlee) [inline career-readiness.md:11]: I suggest adding a bit more to these wireframes than just a heading and background colors

File: code-challenges.test.js [hunk 2]
```diff
@@ -49,8 +92,56 @@ const studyMinutesWeek2 = {
   friday: 15,
   saturday: 65
 }
+const result = sortObjectValues(studyMinutesWeek2)
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
+})
+})
```
Comment (chrisyhlee) [inline code-challenges.test.js:98]: This test has no expect statements

File: code-challenges.test.js [hunk 3]
```diff
@@ -49,8 +92,56 @@ const studyMinutesWeek2 = {
...
+//for each key in object(obj)
+//add the value with the key to values array
+//sort the values array in order
+//return the sorted values in a array
+
+const sortObjectValues = (obj) => {
+  const values = Object.values(obj)
+  values.sort((a, b) => a - b)
+  return values
+}
```
Comment (chrisyhlee) [inline code-challenges.test.js:123]: Your pseudo code doesn't match your function

File: code-challenges.test.js [hunk 4]
```diff
@@ -13,22 +13,63 @@
...
 
 // Pseudo code:
+//create function named generateFibonacci 
+//use if to check length is less than two
+// return "Length should be greater than or equal to 2"
+//create array called fibonacci and add 1 to the beginning
+//use for to check i and the lengths
+//add the sum of the last two numbers in the array
+//add nextFibonacci to the end of the fibonacci array
+//return fibonacci array
```
Comment (chrisyhlee) [inline code-challenges.test.js:40]: Line 37 doesn't make sense, what do you mean check i and the lengths?
Here is how you can rewrite this:
- create variable holding array with 1 and 1 as values
- iterate using for loop starting at index 2
- each iteration adds the values at the previous 2 indices then adds the sum to the created array

**Review by chrisyhlee — APPROVED** [review]
Definitely improving each week, and like you said, keep working on that vocab. I left an example of how you can rewrite pseudo code for challenge 1 to hopefully help you. Other than that, I left some other comments as well. Some pseudo code didn't match the code it was supposed to explain, and your second test doesn't have any expect statements. Make sure to pay attention to that moving forward. This assessment displays Novice.

File: career-readiness.md (hunk 1 above)
Comment (MichaelReyes1) [reply career-readiness.md:11]: been slowly working on this but i got you.

File: code-challenges.test.js (hunk 4 above)
Comment (MichaelReyes1) [reply code-challenges.test.js:40]: yeah i definitely been trying to work on my pseudo code and explaining that better.

File: code-challenges.test.js (hunk 2 above)
Comment (MichaelReyes1) [reply code-challenges.test.js:98]: yeah i missed that somehow

File: code-challenges.test.js (hunk 3 above)
Comment (MichaelReyes1) [reply code-challenges.test.js:123]: copy

**Pull request merged**